h2o-genmodel/src/main/java/hex/genmodel/ModelMojoReader.java

```diff
@@ -263,20 +264,19 @@ else if (line.equals("[columns]")) {
       int n_elements = Integer.parseInt(info[0]);
       String domfile = info[1];
       String[] domain = new String[n_elements];
-      BufferedReader br = _reader.getTextFile("domains/" + domfile);
-      try {
+
+      try (BufferedReader br = _reader.getTextFile("domains/" + domfile)) {
         String line;
         int id = 0;  // domain elements counter
-        while (true) {
-          line = br.readLine();
+        while ((line = br.readLine()) != null) {
           if (line == null) break;
```

Comment:
unnecessry check.

Reply:
Forgot it there. Removed it in https://github.com/h2oai/h2o-3/pull/3554/commits/eaa08fd3f1171dd7c6d492767141404a19743ef0. Thanks.